### DISCUSSION TITLE
SAML SSO setup page URL

### DISCUSSION THREAD

**Discussion by testkoja** [community/community#34184] [Enterprise]
Hi,

I am looking for the URL of the SAML SSO setup page but I cannot find it anywhere.
I guess this is the road: your organization > Settings > Security > SAML SSO

Can someone please help?

**Comment by djdefi** [comment]
To use SAML single sign-on, your organization must be on an GitHub Enterprise Cloud plan. If your organization is on a GitHub Enterprise Cloud plan, and you are still unable to see the settings then I recommend recaching out to [GitHub Support](https://support.github.com/) to take a look!
